packages/realm-server/migrations/1729595943510_billing-adjustments.js

```diff
@@ -0,0 +1,68 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+  pgm.renameTable('credit_balance_events', 'credits_ledger');
+
+  pgm.addColumn('stripe_events', {
+    is_processed: { type: 'boolean', notNull: true, default: false },
+  });
+
+  pgm.alterColumn('subscriptions', 'ended_at', {
+    type: 'timestamp',
+    notNull: false,
+  });
+
+  pgm.alterColumn('subscriptions', 'started_at', {
```

Comment:
Changing timestamp types to unix time so that we can directly sync with Stripe's timestamps which also come as unix time

Reply:
unix times do not need a `bigint`--they can fit comfortably as a normal integer